host-bmc/dbus/custom_dbus.cpp

```diff
@@ -139,5 +139,17 @@ void CustomDBus::implementMotherboardInterface(const std::string& path)
                                 pldm::utils::DBusHandler::getBus(), path));
     }
 }
+
+void CustomDBus::implementObjectEnableIface(const std::string& path, bool value)
+{
+    if (!enabledStatus.contains(path))
+    {
+        enabledStatus.emplace(
+            path, std::make_unique<Enable>(pldm::utils::DBusHandler::getBus(),
+                                           path.c_str()));
+        enabledStatus.at(path)->enabled(value);
+    }
+}
```

Comment:
if the interface is already present ..we cannot use this function to update the property ?

Reply:
Currently we can not use this function to update the property. PLDM just creates "Enable" interface and Enabled property with default value as false.
Do you have any usecase where PLDM needs to update the enabled property?

Reply:
done